index.html

```diff
@@ -1615,6 +1643,10 @@ <h4>Additional Constraints</h4>
             applied to the Form elements of a <a>WoT Thing
               Description</a> conforming to the <a>Core
               profile</a>:
+	  <em>
+	  FIX: this should have an table assertion per row, and this should be informative
+	  or a parent assertion.
+	  </em>
```

Comment:
so... I'm converting this to the markup (it's a table which is a list of assertions as opposed to a set of mandatory fields), but just noticed it says "scopes MUST NOT be used".  Well... I kind of object to that, otherwise you are throwing away a valuable access control feature of OAuth, but I'll leave it for now and bring it up later.